cmd/generate/generate.go

```diff
@@ -33,10 +33,11 @@ import (
 	routeapi "github.com/openshift/origin/pkg/route/api/v1"
 	templateapi "github.com/openshift/origin/pkg/template/api/v1"
 	userapi "github.com/openshift/origin/pkg/user/api/v1"
-	appsapi "k8s.io/kubernetes/pkg/apis/apps/v1alpha1"
 	resourceapi "k8s.io/kubernetes/pkg/api/resource"
 	rapi "k8s.io/kubernetes/pkg/api/unversioned"
 	kapi "k8s.io/kubernetes/pkg/api/v1"
+	appsapi "k8s.io/kubernetes/pkg/apis/apps/v1beta1"
+	batchapi "k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
```

Comment:
Initially is was because the the appsapi used to be v1alpha1 and that had an issue after the upgrade from memory, so I moved that to v1beta1 and followed suit with batch api, just using what I assumed was the correct existing approach.  Also the kubernetes docs used `batch/v2alpha1` in the example https://kubernetes.io/docs/user-guide/cron-jobs/. 

I just tried using `k8s.io/kubernetes/pkg/apis/batch` and `k8s.io/kubernetes/pkg/apis/apps` and there are a ton of compilation errors which I guess there's some packages that have changed or something.

```
[ERROR] /Users/jamesrawlings/work/src/github.com/fabric8io/kubernetes-model/kubernetes-model/target/generated-sources/annotations/AzureDiskVolumeSourceFluent.java:[1,8] <identifier> expected
[ERROR] /Users/jamesrawlings/work/src/github.com/fabric8io/kubernetes-model/kubernetes-model/target/generated-sources/annotations/AzureDiskVolumeSourceFluentImpl.java:[1,8] <identifier> expected
[ERROR] /Users/jamesrawlings/work/src/github.com/fabric8io/kubernetes-model/kubernetes-model/target/generated-sources/annotations/AzureDiskVolumeSourceBuilder.java:[1,8] <identifier> expected
[ERROR] /Users/jamesrawlings/work/src/github.com/fabric8io/kubernetes-model/kubernetes-model/target/generated-sources/annotations/DoneableAzureDiskVolumeSource.java:[1,8] <identifier> expected
[ERROR] /Users/jamesrawlings/work/src/github.com/fabric8io/kubernetes-model/kubernetes-model/target/generated-sources/annotations/PersistentVolumeClaimSpecFluent.java:[1,8] <identifier> expected
```